docs/cheaha.md

```diff
@@ -0,0 +1,32 @@
+# nf-core/configs: Cheaha (UAB HPC) Configuration
+
+All nf-core pipelines have been successfully configured for use on the Cheaha HPC cluster at the [The University of Alabama at Birmingham](https://www.uab.edu/home/).
+
+To use, run the pipeline with `-profile cheaha`. This will download and launch the [`cheaha.config`](../conf/cheaha.config) which has been pre-configured with a setup suitable for the Cheaha HPC cluster. Using this profile, a docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
+
+Before running the pipeline you will need to load Singularity and Nextflow using the environment module system on Cheaha. You can do this by issuing the commands below:
+
+```bash
+## Singularity environment modules
+module purge
+module load Singularity
+module load Nextflow
+```
+
+All of the intermediate files required to run the pipeline will be stored in the `work/` directory. It is recommended to delete this directory after the pipeline has finished successfully because it can get quite large, and all of the main output files will be saved in the `results/` directory anyway.
```

Comment:
FYI: you could also consider adding `cleanup = true` to the profile to have this to auto clean up.

Reply:
Thanks for the suggestion @jfy133 . I think we will keep this option off since we find some components of the pipeline in the `work` directory useful for testing pipelines and debugging when needed, and others may benefit from it too without auto-deletion. So a more curated approach to this directory may be overall more beneficial.